product_sticker/readme/USAGE.rst

```diff
@@ -0,0 +1,7 @@
+You can add Stickers to Attributes resulting in different behaviours:
+- If an Image has not Company, it will be available to all Companies
+- If an Image has not Attribute, it will be available to all Attributes
+- If an Image has not Attribute Value, it will be available to all Attribute Values of the Attribute
+- If an Image has not Category, it will be available to all Categories
```

Comment:
```suggestion
- If an Image has no Company, it will be available to all Companies
- If an Image has no Attribute, it will be available to all Attributes
- If an Image has no Attribute Value, it will be available to all Attribute Values of the Attribute
- If an Image has no Category, it will be available to all Categories
```